src/assets/config.json

```diff
@@ -4,13 +4,15 @@
     "agenciesURL": "Agencies",
     "citationURL": "Citations",
     "errorsURL": "Errors",
+    "gageURL": "https://test.streamstats.usgs.gov/gagestatsservices/",
```

Comment:
Since the gageStatsBaseURL was added earlier, you can remove this line. You can do a full Find and Replace in VSCode.